projects/pulsar_adc/Readme.md

```diff
@@ -65,4 +65,11 @@ Here are some pointers to help you:
   * AD4010
   * AD4011
   * ADAQ4003
-  * AD4020
\ No newline at end of file
+  * AD4020
+
+build example:
+make FMC_N_PMOD=0 - builds PMOD version
```

Comment:
It's a bit confusing that setting the `PMOD` option to 0 builds the PMOD version (it is easy to miss the `N` or not know what it means, espeically when it gets squished in `FMCNPMOD1_SPIOPMODE2`). Could we change it so that `FMC_PMOD=1` selects the PMOD version?